_sections/inf4/adbs.md

```diff
@@ -1,6 +1,9 @@
 ---
 year: inf4
 title: ADBS
+course-acronym: adbs
+archived: false
```

Comment:
i would exclude this field when it's false, since it's a default. up to you!

Reply:
Well, I've put it in nearly every file now, so I'm not removing the damned thing!